src/Localization/en/MaterializedViewsBuilder.json

```
@@ -14,35 +14,58 @@
     "CosmosD32s": "Cosmos.D32s (General Purpose Cosmos Compute with 32 vCPUs, 128 GB Memory)",
-    "CreateMessage": "MaterializedViewsBuilder resource is being created.",
+    "CreateMessage": "Materialized views builder resource is being created.",
     "CreateInitializeTitle": "Provisioning resource",
-    "CreateInitializeMessage": "Materializedviews Builder resource will be provisioned.",
+    "CreateInitializeMessage": "Materialized views Builder resource will be provisioned.",
     "CreateSuccessTitle": "Resource provisioned",
-    "CreateSuccesseMessage": "Materializedviews Builder resource provisioned.",
+    "CreateSuccesseMessage": "Materialized views Builder resource provisioned.",
     "CreateFailureTitle": "Failed to provision resource",
-    "CreateFailureMessage": "Materializedviews Builder resource provisioning failed.",
-    "UpdateMessage": "MaterializedViewsBuilder resource is being updated.",
+    "CreateFailureMessage": "Materialized views Builder resource provisioning failed.",
+    "UpdateMessage": "Materialized views builder resource is being updated.",
     "UpdateInitializeTitle": "Updating resource",
-    "UpdateInitializeMessage": "Materializedviews Builder resource will be updated.",
+    "UpdateInitializeMessage": "Materialized views Builder resource will be updated.",
     "UpdateSuccessTitle": "Resource updated",
-    "UpdateSuccesseMessage": "Materializedviews Builder resource updated.",
+    "UpdateSuccesseMessage": "Materialized views Builder resource updated.",
     "UpdateFailureTitle": "Failed to update resource",
-    "UpdateFailureMessage": "Materializedviews Builder resource updation failed.",
-    "DeleteMessage": "MaterializedViewsBuilder resource is being deleted.",
+    "UpdateFailureMessage": "Materialized views Builder resource updation failed.",
+    "DeleteMessage": "Materialized views builder resource is being deleted.",
     "DeleteInitializeTitle": "Deleting resource",
-    "DeleteInitializeMessage": "Materializedviews Builder resource will be deleted.",
+    "DeleteInitializeMessage": "Materialized views Builder resource will be deleted.",
     "DeleteSuccessTitle": "Resource deleted",
-    "DeleteSuccesseMessage": "Materializedviews Builder resource deleted.",
+    "DeleteSuccesseMessage": "Materialized views Builder resource deleted.",
     "DeleteFailureTitle": "Failed to delete resource",
-    "DeleteFailureMessage": "Materializedviews Builder resource deletion failed.",
+    "DeleteFailureMessage": "Materialized views Builder resource deletion failed.",
     "ApproximateCost": "Approximate Cost Per Hour",
-    "CostText": "Hourly cost of the Materializedviews Builder resource depends on the SKU selection, number of instances per region, and number of regions.",
+    "CostText": "Hourly cost of the materialized views Builder resource depends on the SKU selection, number of instances per region, and number of regions.",
```

Is this true? I thought the builder was only provisioned in one region. So cost would only depend on SKU and number of nodes.

```suggestion
    "CostText": "Hourly cost of the materialized views builder resource depends on the SKU selection, number of instances per region, and number of regions.",
```